config/locales/plurals.rb

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
The commit message is missing a definition for `[1]`

Reply:
Doh, have added the definition.